common/src/WooCommerce.php

```diff
@@ -440,9 +440,15 @@ private function getExistingComment(int $post_id, string $author_email, int $rev
     private function getCommentData(\SimpleXMLElement $review) {
         $pf = new WC_Product_Factory();
         $product_id = (int) $review->products->product->external_id;
-        if (!$pf->get_product($product_id)) {
+
+        if (!$product = $pf->get_product($product_id)) {
             throw new RuntimeException(sprintf("No product with ID {$product_id}"));
         }
+
+        if ($product->get_parent_id() !== 0) {
```

Comment:
Wouldn't trust the types returned, and check for truthiness instead, so that `null`, etc., are also ignored:

```suggestion
        if ($product->get_parent_id()) {
```